custom-namespaces.yaml

```diff
@@ -392,6 +392,25 @@ revenue:
       type: table_view
       tables:
         - table: mozdata.stripe.itemized_payout_reconciliation
+revenue_private:
+  glean_app: false
+  connection: bigquery-oauth
+  pretty_name: Revenue Private
+  owners:
+    - akomarzewski@mozilla.com
+    - skahmann@mozilla.com
+    - xluo@mozilla.com
+  spoke: looker-spoke-private
+  views:
+    revenue_forecast_review_2022:
+      type: table_view
+      tables:
+        - table: mozdata.revenue_cat3_analysis.revenue_forecast_review_long
+  views:
+    revenue_budget_2022:
+      type: table_view
+      tables:
+        - table: mozdata.revenue_cat3_analysis.budget_complete_2022       
```

Comment:
lookml-generator has been failing since the service account doesn't have access to get the table schema: 

```
2022-10-13 17:54:21,467] {{pod_launcher.py:149}} INFO -   File "/usr/local/lib/python3.8/site-packages/google/api_core/retry.py", line 286, in retry_wrapped_func
[2022-10-13 17:54:21,467] {{pod_launcher.py:149}} INFO -     return retry_target(
[2022-10-13 17:54:21,467] {{pod_launcher.py:149}} INFO -   File "/usr/local/lib/python3.8/site-packages/google/api_core/retry.py", line 189, in retry_target
[2022-10-13 17:54:21,468] {{pod_launcher.py:149}} INFO -     return target()
[2022-10-13 17:54:21,468] {{pod_launcher.py:149}} INFO -   File "/usr/local/lib/python3.8/site-packages/google/cloud/_http/__init__.py", line 494, in api_request
[2022-10-13 17:54:21,468] {{pod_launcher.py:149}} INFO -     raise exceptions.from_http_response(response)
[2022-10-13 17:54:21,468] {{pod_launcher.py:149}} INFO - google.api_core.exceptions.Forbidden: 403 GET https://bigquery.googleapis.com/bigquery/v2/projects/mozdata/datasets/revenue_cat3_analysis/tables/budget_complete_2022?prettyPrint=false: Access Denied: Table mozdata:revenue_cat3_analysis.budget_complete_2022: Permission bigquery.tables.get denied on table mozdata:revenue_cat3_analysis.budget_complete_2022 (or it may not exist).
```
This should just be a matter of configuring the permissions to the dataset correctly, maybe @akkomar  can look into this.

Reply:
This isn't a dataset misconfiguration per se; rather it's the current policy that `_analysis` datasets aren't appropriate for production workloads and so airflow doesn't have direct access to them. When we've needed to expose such datasets to Looker (just once for a search terms dataset that I can recall), we have granted table-level access with the caveat that the data shouldn't be treated as production as it's developer-managed.

Reply:
@whd what do we need to do in order to grant Airflow access to this table? @xluo-ds is ok [with managing and monitoring it](https://mozilla.slack.com/archives/C4D5ZA91B/p1665687383432039?thread_ts=1665509556.496149&cid=C4D5ZA91B).

Reply:
Now that this has happened twice I think the Platform Infra WG should consider formalizing the conditions under which we allow special cases for Looker/Airflow access to developer-managed tables. For instance we might require that tables or dashboards conform to some naming convention that indicates that the data isn't managed in a production capacity. This could be as simple as socializing that anything with `_analysis` as a dataset suffix is not managed by DE/DSRE.

For now though I will file a terraform PR to grant this access to unblock this work.

Reply:
@whd  is it done? Wonder whether I can continue

Reply:
yes, the view is availble in looker-hub:https://github.com/mozilla/looker-hub/blob/main/revenue_private/views/revenue_budget_2022.view.lkml

Reply:
@scholtzan  what about the other one? I seem not see it https://github.com/mozilla/looker-hub/tree/main/revenue_private/views

Reply:
Ah I overlooked the typo in the config. Pushing a fix now

Reply:
@scholtzan  thank you :)

Reply:
it's all there now: https://github.com/mozilla/looker-hub/tree/main/revenue_private/views

Reply:
@scholtzan see it now, thank you!